genesis-tools/genesis-csv-to-json/src/csv_to_json_configs.rs

```diff
@@ -38,7 +38,7 @@ pub fn csv_to_json_configs(home: &Path, chain_id: String, tracked_shards: Vec<Sh
     // Verify that key files exist.
     assert!(home.join(NODE_KEY_FILE).as_path().exists(), "Node key file should exist");
 
-    if tracked_shards.iter().any(|shard_id| *shard_id >= NUM_SHARDS) {
+    if tracked_shards.iter().any(|&shard_id| shard_id >= NUM_SHARDS) {
```

Comment:
this should check if every tracked shard_id is in the shard_ids vector

Reply:
Hmm... I wasn't sure what to do here.  We have only defined `NUM_SHARDS` here and not a vector of shard ids.  I think in a separate change, we can have a global vector of shard ids instead of NUM_SHARDS and that I will clean things up more.  WDYT?

Reply:
I just realized it's outside of the regular node operation. Yeah it's hard to tell what's the num shards without access to chain, epoch and the current shard layout. I'm totally fine leaving it as is. Do you know where is this genesis-tool used?

Reply:
No idea what the purpose of this tool is sadly.